contracts/pair_stable/schema/raw/execute.json

```diff
@@ -0,0 +1,324 @@
+{
```

Comment:
Remove all the json schema files and add them to gitignore

Reply:
done